integration-tests/src/test/java/oracle/kubernetes/operator/JrfInOperatorTest.java

```diff
@@ -0,0 +1,141 @@
+// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import oracle.kubernetes.operator.utils.JRFDomain;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.OracleDB;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Simple JUnit test file used for testing Operator for JRF domains.
+ *
+ * <p>This test is used for creating Operator(s) and multiple JRF domains which are managed by the
+ * Operator(s).
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class JrfInOperatorTest extends BaseTest {
+
+  // property file used to customize operator properties for operator inputs yaml
+
+  private static String operator1File = "jrfoperator1.yaml";
+
+  // file used to customize domain properties for domain, PV and LB inputs yaml
+  private static String jrfdomainonpvwlstFile = "jrfdomainonpvwlst.yaml";
+
+  // property file for oracle db information
+  private static String dbPropsFile = "oracledb.properties";
+
+  // property file used to configure constants for integration tests
+  private static String appPropsFile = "OperatorIT.properties";
```

Comment:
you can use APP_PROPS_FILE from BaseTest

Reply:
pushed another commit to jrf-in-op-from-develop